blueprints/component/glimmer-files/__root__/__path__/__name__.ts

```diff
@@ -0,0 +1,5 @@
+import Component from '@glimmer/component';
+
+interface Args {}
```

Comment:
After thinking on it more—let's go with `<% classifiedModuleName %>Args` here and below, because I think `MyComponentArgs` (while annoyingly long in some ways) is probably the best *default* given how editor imports and the like work—the heuristics you get with VS Code and the TS language server are pretty good, so people can rename as they prefer, but this way autocomplete and the like have the best changes of resolving to what you want across *all* editors.

Reply:
Done!